spring-test/src/test/java/org/springframework/test/web/servlet/htmlunit/HostRequestMatcherTests.java

```diff
@@ -31,14 +31,14 @@
 	public void localhost() throws Exception {
 		WebRequestMatcher matcher = new HostRequestMatcher("localhost");
 		assertMatches(matcher, "http://localhost/jquery-1.11.0.min.js");
-		assertDoesNotMatch(matcher, "http://example.com/jquery-1.11.0.min.js");
+		assertDoesNotMatch(matcher, "http://company.example/jquery-1.11.0.min.js");
```

Comment:
Shouldn't that use `https`?

Reply:
The test is validating another http URL does not match, so we need http here. In that case we can use See https://github.com/spring-io/nohttp/tree/master/nohttp#thought-process